tests/commands/timezone.py

```diff
@@ -154,5 +155,47 @@ def runTest(self):
         self.assert_parse_error("timezone Europe/Sofia --nontp --ntpservers=0.fedora.pool.ntp.org,1.fedora.pool.ntp.org")
 
 
+class F32_TestCase(F25_TestCase):
+    command = "timezone"
+
+    def runTest(self):
+
+        # Failures
+        # unknown argument
+        self.assert_parse_error("timezone --blah")
+        # more than two timezone specs
+        self.assert_parse_error("timezone foo bar", KickstartParseError,
+                                'One or zero arguments are expected for the timezone command')
+        self.assert_parse_error("timezone --utc foo bar", exception=KickstartParseError)
+        # no options
+        self.assert_parse_error("timezone", KickstartParseError,
+                    'At least one option and/or an argument are expected for the timezone command')
+        # contradictory options
+        self.assert_parse_error(
+            "timezone Europe/Sofia --nontp --ntpservers=0.fedora.pool.ntp.org,1.fedora.pool.ntp.org")
+
+        # Successes
+        # normal contents
+        self.assert_parse("timezone Europe/Prague --ntpservers=ntp.cesnet.cz",
+                          "timezone Europe/Prague --ntpservers=ntp.cesnet.cz\n")
+        # no timezone spec
+        self.assert_parse("timezone --ntpservers=ntp.cesnet.cz",
+                          "timezone --ntpservers=ntp.cesnet.cz\n")
+
+        # New in F32: any variant of UTC should be returned as --utc (again - was in FC3)
+        with warnings.catch_warnings(record=True) as w:
```

Comment:
You should be able to use `self.assertWarns` to check the warnings. See: https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertWarns